db/migrate/20200416154923_add_logo_key_to_service_provider.rb

```diff
@@ -0,0 +1,5 @@
+class AddLogoKeyToServiceProvider < ActiveRecord::Migration[5.2]
+  def change
+    add_column :service_providers, :logo_key, :string
```

Comment:
can we call this like `remote_logo_key` or something? Some sort of hint that this outside the database? Or does this need to be the same column as in Dashboard, etc

Reply:
It needs to be the same as in the JSON published by the dashboard. That said, it's not too late to change the dashboard...

Reply:
Thanks!! Appreciate it